IFC_Adapter/Methods/Modify/CopyProperties.cs

```diff
@@ -0,0 +1,58 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Adapters.IFC.Properties;
+using BH.oM.Base;
+using System.Collections.Generic;
+using Xbim.Ifc2x3.Kernel;
+using Xbim.Ifc2x3.PropertyResource;
+
+namespace BH.Adapter.IFC
+{
+    public static partial class Modify
+    {
+        /***************************************************/
+        /****              Public methods               ****/
+        /***************************************************/
+
+        public static void CopyParameters(this IBHoMObject bHoMObject, IfcObject element)
+        {
+            if (bHoMObject == null || element == null)
+                return;
+
+            List<BH.oM.Adapters.IFC.Properties.IfcProperty> properties = new List<BH.oM.Adapters.IFC.Properties.IfcProperty>();
```

Comment:
Consider adding `using BH.oM.Adapters.IFC.Properties` for readability

Reply:
I left it this way because of ambiguity between `BH.oM.Adapters.IFC.Properties.IfcProperty` and `Xbim.Ifc2x3.PropertyResource.IfcProperty`. In general, this could be a smart move to rename the BHoM object to something else as this is likely to occur on every interaction with IFC properties. However, I could not think of any name that would be different yet close enough. Any ideas?

Reply:
Leave as is